pages/add-user-page.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useState } from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  useColorScheme,
+  View,
+} from 'react-native';
+import { Colors } from 'react-native/Libraries/NewAppScreen';
+import { signinValidator } from '../regex';
+import { Section } from '../section';
+import { Styles } from '../styles';
+
+export const AddUserPage = () => {
+  const isDarkMode = useColorScheme() === 'dark';
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
+
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [phone, setPhone] = useState('');
+  const [birthDate, setBirthDate] = useState('');
+  const [password, setPassword] = useState('');
+  const [role, setRole] = useState("");
+
+  const [nameError, setNameError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [phoneError, setPhoneError] = useState(false);
+  const [birthDateError, setBirthDateError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
+  const [roleError, setRoleError] = useState(false);
```

Comment:
Esses estados são boolenas, mas pela nomenclatura tive impressão que seriam strings. Talvez usar o prefixo `is`?

Reply:
Solucionado na branch `feature/styledLoginAddUsers`